pkg/bindings/images/build.go

```diff
@@ -682,8 +682,15 @@ func nTar(excludes []string, sources ...string) (io.ReadCloser, error) {
 						}
 					}
 				}
-				name := filepath.ToSlash(strings.TrimPrefix(path, s+separator))
-
+				var name string
+				if i == 0 {
+					name = filepath.ToSlash(strings.TrimPrefix(path, s+separator))
+				} else {
+					if !d.Type().IsRegular() {
+						return fmt.Errorf("path %s must be a regular file", path)
+					}
+					name = filepath.ToSlash(path)
```

Comment:
This is the absolute path of the file, which can't be right.

Reply:
Is this resolved, or is a change still needed? @rhatdan

Reply:
I convinced Nalin that it is correct (Well at least it works.)